platform/view/services/storage/driver/sql/query/pagination/save_load.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package pagination
+
+import (
+	"github.com/hyperledger-labs/fabric-smart-client/platform/common/driver"
+)
+
+func SavePagination(p driver.Pagination) ([]byte, error) {
```

Comment:
why don't add directly to `driver.Pagination` a function to `Serialize` it? 
This way, we don't have to modify function every time we introduce a new implementation of the interface.